exotations/solvers/exotica_ik_solver/include/exotica_ik_solver/ik_solver.h

```diff
@@ -59,15 +56,12 @@ class IKSolver : public MotionSolver, public Instantiable<IKSolverInitializer>
 private:
     IKSolverInitializer parameters_;
 
-    Eigen::MatrixXd PseudoInverse(Eigen::MatrixXdRefConst J);
     void ScaleToStepSize(Eigen::VectorXdRef xd);
 
     UnconstrainedEndPoseProblem_ptr prob_;  // Shared pointer to the planning problem.
 
-    Eigen::MatrixXd Cinv_;  //!< Weight Matrices
-    Eigen::MatrixXd C_;
-    Eigen::MatrixXd W_;
-    Eigen::MatrixXd Winv_;
+    Eigen::MatrixXd C_;  //!< \brief Regularisation (use values from interval <0, 1))
```

Comment:
This is wrong Doxygen. Should be `///<` (fixed the same issue yesterday in other places).

Reply:
This should be fine. I've used similar documentation strings before.
See: http://www.doxygen.nl/manual/docblocks.html (Putting documentation after members
, 3rd box).